bolt/lib/Rewrite/LinuxKernelRewriter.cpp

```diff
@@ -248,6 +248,9 @@ class LinuxKernelRewriter final : public MetadataRewriter {
   /// Update ORC data in the binary.
   Error rewriteORCTables();
 
+  /// Validate written ORC tables after binary emission.
+  Error validateORCTables();
```

Comment:
If we can split the validation in its own commit, that would be good.

Reply:
Sure, if it helps you to review.